src/code/z_actor.c

```diff
@@ -2707,19 +2708,88 @@ void Actor_DrawLensActors(PlayState* play, s32 numInvisibleActors, Actor** invis
+ * allowing actors in the immediate vicinty behind and to the sides of the player to be detected.
+ *
+ * This function returns true if the actor is within bounds, false if not.
+ * The comparison is done in projective space against the actor's projected position as the viewing frustum
+ * in world space transforms to a box in projective space, making the calculation easy.
+ *
+ * Every actor can set properites for their own culling volume, changing it's dimensions to suit the needs of it and
+ * it's environment. Note that the following properties are projective properties as they are compared
+ * with the actor's position after perspective projection is applied. The units are therefore not directly comparable to
+ * world units. These depend on the current view parameters (fov, aspect, scale, znear, zfar). The default parameters
+ * considered are (60 degrees, 4/3, 1.0, 10, 12800).
+ *
+ *    cullingVolumeDistance: Configures how far forward the far plane of the frustum should extend.
+ *                           This along with cullingVolumeScale determines the maximum distance from
+ *                           the camera eye that the actor can be detected at. This quantity is related
+ *                           to world units by a factor of
+ *                                   (znear - zfar) / ((znear + zfar) * scale).
+ *                           For default view parameters, increasing this property by 1 increases the
+ *                           distance by ~0.995 world units.
+ *
+ *    cullingVolumeScale: Scales the entire culling volume in all directions except the downward
+ *                        direction. Both the frustum and the box will scale in size. This quantity is
+ *                        related to world units by different factors based on direction:
+ *                         - For the forward and backward directions, they are related in the same way
+ *                           as above. For default view parameters, increasing this property by 1 increases
+ *                           the forward and backward scales by ~0.995 world units.
+ *                         - For the sideways directions, the relation to world units is
+ *                                   (aspect / scale) * tan(0.5 * fov)
+ *                           For default view parameters, increasing this property by 1 increases the
+ *                           sideways scales by ~0.77 world units.
+ *                         - For the upward direction, the relation to world units is
+ *                                   (1 / scale) * tan(0.5 * fov)
+ *                           For default view parameters, increasing this property by 1 increases the
+ *                           scale by ~0.58 world units.
+ *
+ *    cullingVolumeDownward: Sets the height of the culling volume in the downward direction. Increasing
+ *                           this value will make actors below the player more easily detected. This
```

Comment:
```suggestion
 *                           this value will make actors below the camera more easily detected. This
```

Reply:
Same reasoning here, I think its fine to say player for this one

Reply:
Nevermind, I will change it in all places